extensions/aws2-ddb/runtime/src/doc/configuration.adoc

```diff
@@ -0,0 +1,60 @@
+There are 2 ways of DynamoDB client creation:
+
+=== Client created by this extension
+
+In that case fill following properties:
+
+
+[source,properties]
+----
+camel.component.aws2-ddb.access-key=${AWS_ACCESS_KEY}
+camel.component.aws2-ddb.secret-key=${AWS_SECRET_KEY}
+camel.component.aws2-ddb.region=${AWS_REGION:us-east-1}
+----
+
+or
+
+[source,properties]
+----
+camel.component.aws2-ddbstream.access-key=${AWS_ACCESS_KEY}
+camel.component.aws2-ddbstream.secret-key=${AWS_SECRET_KEY}
+camel.component.aws2-ddbstream.region=${AWS_REGION:us-east-1}
+----
+
+=== Client created by `quarkus-amazon-dynamodb`
+
+o use client created by `quarkus-amazon-dynamodb` extension, follow the  https://quarkus.io/guides/amazon-dynamodb#configuring-dynamodb-clients[documentation] with some restrictions:
+
+- Apache client type has to be selected
+[source,properties]
+----
+quarkus.dynamodb.sync-client.type=apache
+----
+
+- All required dependencies are already part of the extension. There is no need to add them.
```

Comment:
If there is no need to add them, maybe we don't show the example below ?

Reply:
That makes sense, I'll remove it immediately.

Reply:
done